data/spells/scripts/attack/avalanche.lua

```diff
@@ -7,9 +7,9 @@ local area = createCombatArea(AREA_CIRCLE3X3)
 setCombatArea(combat, area)
 
 function onGetFormulaValues(cid, level, maglevel)
-	min = -((level / 5) + (maglevel * 1.2) + 7)
-	max = -((level / 5) + (maglevel * 2.85) + 16)
-	return min, max
+	min = (level / 5) + ((maglevel * 1.2) + 7)
+	max = (level / 5) + ((maglevel * 2.85) + 16)
+	return -min, -max
 end
```

Comment:
Oh yeah, I do prefer the more parenthesized way, it makes much clearer to see where each factor influences. I think the first form I suggested is good because it splits the level and the magic level part out of the minimum damage value :P